sed/loader/flash/dataframe.py

```diff
@@ -0,0 +1,258 @@
+from __future__ import annotations
+
+import h5py
+import numpy as np
+from pandas import concat
+from pandas import DataFrame
+from pandas import Index
+from pandas import MultiIndex
+from pandas import Series
+
+from sed.loader.flash.utils import get_channels
+
+
+class DataFrameCreator:
+    """
+    Utility class for creating pandas DataFrames from HDF5 files with multiple channels.
+    """
+
+    def __init__(self, config_dataframe: dict, h5_file: h5py.File) -> None:
```

Comment:
There used to be a problem with the abc class inheritance before. But it seems this problem is now gone since i split the classes. I will make an update